docs/content/direct/combined-status.md

```diff
@@ -8,38 +8,85 @@ It may make sense to expand the latter into a general feature in the future, acc
 
 The general technique involves the following ideas.
 
-1. The way that reported state is combined is specified by the user, in a simple but powerful way modeled on SQL. This is chosen because it is a well worked out set of ideas, is widely known, and is something that we may someday want to use in our implementation. We do not need to support anything like full SQL (for any version of SQL). This proposal only involves one particular pattern of relatively simple SELECT statement, and a small subset of the expression language.
+1. The way that reported state is combined is specified by the user, in a simple but powerful way modeled on SQL. This is chosen because it is a well worked out set of ideas, is widely known, and is something that we may someday want to use in our implementation. We do not need to support anything like full SQL (for any version of SQL). This proposal only involves one particular pattern of relatively simple SELECT statement, and a different expression language (one of growing currency in the Kubernetes milieu).
 
-1. The specification of how to combine reported state is defined in `StatusCollector` objects. These objects are referenced in the BindingPolicies right next to the criteria for selecting workload objects. This saves users the trouble of having to write selection criteria twice. With the specification being separate, it is possible to have a library of `StatusCollectors` that can be reused across different BindingPolicies. In the future KubeStellar would provide a library of `StatusCollector` objects that cover convenient use-cases for kubernetes built-in resources such as deployments.
+1. The expressions may reference the content of the workload object as it sits in the WDS as well as the state returned from the WECs.
```

Comment:
I guess the as well as refers to the "content ..."   ?

Reply:
I am not sure I understand the question. "as well as" is not a reference, it is a joiner.